libs/explored-js/src/api.ts

```diff
@@ -56,6 +56,14 @@ import {
   MetricsByIDParams,
   MetricsByIDPayload,
   MetricsByIDResponse,
+  OutputSiacoineParams,
```

Comment:
Looks like an extra e in these: Siacoine

Reply:
I went ahead and fixed this. I was just moving around the imports so they'd be in more of an alphabetical order. Do you think this needs included in the changeset?

Reply:
In the future yes since consumer of the libraries should be aware, but for now while this is fast changing alpha code its ok to skip the tiny stuff and just make sure the changelog covers the main work.

Reply:
Makes sense. Thank you!